queries.sql

```diff
@@ -10,3 +10,52 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
 
+/* Transaction unspecified species*/
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+/* Transaction update animals species */
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
```

Comment:
- [ ] It will be nice to verify if the change was really persisted.

See example:  `SELECT species from animals; -- verify that change persists after commit`

Reply:
Nice tip, I will add it.